Week3Project/Week3Project/users.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Week3Project
+{
+    class User
+    {
+        public string firstName { get; set; }
```

Comment:
Property names and method names, especially public ones, are usually named in PascalCase instead of camelCase in most C# naming conventions